specification/resources/resource-manager/Microsoft.Authorization/stable/2023-01-01/privateLinks.json

```diff
@@ -0,0 +1,409 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PrivateLinksClient",
```

Comment:
This client is PolicyClient in 2020-05-01 version. I remember last time we had to change client(https://github.com/Azure/azure-sdk-for-net/pull/27035) for the SDK. Does this look right to you TJ?